SigProfilerMatrixGenerator/scripts/vcfToBedpe.py

```diff
@@ -0,0 +1,395 @@
+            '%0.2f' % self.qual,
+            self.filter,
+            self.get_info_string(),
+            self.get_format_string(),
+            '\t'.join(self.genotype(s).get_gt_string() for s in self.sample_list)
+        ]))
+        return s
+
+class Genotype(object):
+    def __init__(self, variant, sample_name, gt):
+        self.format = dict()
+        self.variant = variant
+        self.set_format('GT', gt)
+
+    def set_format(self, field, value):
+        if field in [i.id for i in self.variant.format_list]:
+            self.format[field] = value
+            if field not in self.variant.active_formats:
+                self.variant.active_formats.append(field)
+                # sort it to be in the same order as the format_list in header
+                self.variant.active_formats.sort(key=lambda x: [f.id for f in self.variant.format_list].index(x))
+        # else:
+        #     sys.stderr.write('\nError: invalid FORMAT field, \"' + field + '\"\n')
+        #     exit(1)
+
+    def get_format(self, field):
+        return self.format[field]
+
+    def get_gt_string(self):
+        g_list = list()
+        for f in self.variant.active_formats:
+            if f in self.format:
+                if type(self.format[f]) == float:
+                    g_list.append('%0.2f' % self.format[f])
+                else:
+                    g_list.append(self.format[f])
+            else:
+                g_list.append('.')
+        return ':'.join(map(str,g_list))
+
+
+# primary function
+def vcfToBedpe(vcf_path, output_path):
+
+
+    sample = vcf_path.split("/")[-1].split(".")[0]
+
+    vcf_file = open(vcf_path, "r")
+    bedpe_out = open(output_path + sample + ".bedpe.tsv", "w")
+
+    vcf = Vcf()
+    in_header = True
+    sample_list = []
+
+    for line in vcf_file:
+        if in_header:
+            if line[0] == '#':
+                if line[1] != '#':
+                    sample_list = line.rstrip().split('\t')[9:]
+                continue
+            else:
+                # print header
+                headerVal = ['#CHROM_A',
+                           'START_A',
+                           'END_A',
+                           'CHROM_B',
+                           'START_B',
+                           'END_B',
+                           'ID',
+                           'QUAL',
+                           'STRAND_A',
+                           'STRAND_B',
+                           'TYPE',
+                           'FILTER',
+                           'INFO']
+ 
+                if len(sample_list) > 0:
+                    headerVal.append('FORMAT')
+                    headerVal.extend(sample_list)
+
+                bedpe_out.write('\t'.join(headerVal) + '\n')
+                    
+                in_header = False
+
+        v = line.rstrip().split('\t')
+        var = Variant(v, vcf)
+
+        chrom2 = var.chrom
+        b1 = var.pos
+        name = v[2]
+        if var.info['SVTYPE'] != 'BND':
```

Comment:
This relies on `SVTYPE` field which is deprecated in Variant Call Format version 4.4. Your code is already obsolete.

Reply:
We will not be adding support for VCF 4.4 and 4.5 in this release. It will need to be added in a future release. @azhark2, is this something that you could oversee the development on for a future release?